binding/web/src/rhino_worker.ts

```diff
@@ -281,13 +201,28 @@ export class RhinoWorker {
       }
     );
 
+    if (
+      context.customWritePath == null ||
+      context.customWritePath == undefined
+    ) {
+      context.customWritePath = 'rhino_context';
+    }
+    const contextPath = await loadModel(context);
+    const { sensitivity = 0.5 } = context;
+
+    if (model.customWritePath == null || model.customWritePath == undefined) {
+      model.customWritePath = 'rhino_model';
+    }
+    const modelPath = await loadModel(model);
+
     worker.postMessage({
       command: 'init',
       accessKey: accessKey,
-      modelPath: modelPath,
       contextPath: contextPath,
+      sensitivity: sensitivity,
       wasm: this._wasm,
       wasmSimd: this._wasmSimd,
+      modelPath: modelPath,
       options: options,
     });
```

Comment:
this is what I thought it would work, also a suggestion:
```
worker.postMessage({
      command: 'init',
      accessKey: accessKey,
      context: { ...context, base64: '' },
      sensitivity: sensitivity,
      wasm: this._wasm,
      wasmSimd: this._wasmSimd,
      model: { ...model, base64: '' },
      options: options,
    });
```
this way we can just use one create function